### PR TITLE
Adding a progress bar for CLOUD BUILD progress

### PR DESCRIPTION
**Pull request by blckmn**
Also adding language support for the various levels of status and indicating to the user if the failure is a timeout (they should retry) or a failed build (they should check the log).

### CODE REVIEW

**Review by haslinghuis — APPROVED** [review]

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> PASS
 - cooling off period lapsed -> FAIL
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> PASS
 - `Tested` label found -> PASS
 - assigned to an approver -> PASS
 - approver count at least three -> FAIL

**Comment by nerdCopter** [comment]
it works for me
image
image

**Pull request merged**
